docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`9145` Deposits and withdrawals from Hyperliquid will be decoded.
```

Comment:
In the next PR also explain that balances deposited in hyperliquid will now be autodetected